src/tests/Microsoft.Diagnostics.NETCore.Client/GetPublishedProcessesTests.cs

```diff
@@ -77,5 +77,21 @@ public void MultiplePublishedProcessTest()
                 runner[i].Stop();
             }
         }
+
+        [Fact]
+        public void CheckSpecificProcessTest()
+        {
+            TestRunner runner = new TestRunner(CommonHelper.GetTraceePath(), output);
+            runner.Start(3000);
```

Comment:
Consider folding the CheckTransport logic into the TestRunner class itself; it does things like check for paths to make sure that the ipc is there.

Reply:
The point of this test is to explicitly check that the CheckTransport method will return true for a targeted process if it has the diagnostics pipe. I don't want to abstract away the purpose of the test into the test framework code.